recipes/msmpi-bin/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set version="7.1.12437.25" %}
+
+package:
+  name: msmpi-bin
+  version: {{ version }}
+
+source:
+  fn: MSMpiSetup.exe
```

Comment:
I am digressing here but I wonder if we should have some sort of "flag" or build string to identify re-packages. Maybe it is not worth the effort.